packages/backend/src/models/CatalogModel/utils/index.ts

```diff
@@ -19,7 +19,7 @@ export const convertExploresToCatalog = (
         const dimensionsAndMetrics = [
             ...Object.values(baseTable?.dimensions || {}),
             ...Object.values(baseTable?.metrics || {}),
-        ];
+        ].filter((f) => !f.hidden); // Filter out hidden fields from catalog
```

Comment:
thanks @rephus for the explanation - here we remove these on `index`, so when the user compiles the project/refreshes dbt